bin/syclcc-clang

```diff
@@ -571,7 +631,7 @@ class hip_invocation:
         "-isystem", self._clang_include_path,
```

Comment:
This is related to ROCm setup, and is also a workaround, so maybe pull into cxx flags?